administrator/components/com_admin/sql/updates/mysql/4.1.0-2021-06-27.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO `#__extensions` (`package_id`, `name`, `type`, `element`, `folder`, `client_id`, `enabled`, `access`, `protected`, `locked`, `manifest_cache`, `params`, `custom_data`, `checked_out`, `checked_out_time`, `ordering`, `state`) VALUES
+(0, 'plg_system_addmodulebutton', 'plugin', 'addmodulebutton', 'system', 0, 1, 1, 0, 1, '', '', '', 0, NULL, 0, 0);
```

Comment:
```suggestion
(0, 'plg_system_addmodulebutton', 'plugin', 'addmodulebutton', 'system', 0, 1, 1, 0, 1, '', '', '', NULL, NULL, 0, 0);
```
The default value for the `checked_out` column has changed from zero to NULL. Maybe you have copied your SQL statement from an update SQL script which runs before the one which made that change (4.0.0-2020-05-29.sql) so still zero was used.

But here you can use NULL for the `checked_out` column.

Reply:
It's not a big mistake, it would also work with zero. The rest of the recent SQL changes for adding that plugin to new installations and on updates looks ok to me.

Reply:
Thank you! This is very helpful. I will remember this for the next time